BuildTools.sln

```diff
@@ -49,7 +49,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{76ABA507
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RepoTasks", "build\tasks\RepoTasks.csproj", "{F3CAD1F9-68DB-49EA-88A7-573AE7FFE047}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.Tasks", "src\NuGet.Tasks\NuGet.Tasks.csproj", "{020ED083-4076-4711-A52B-2F89EA884F9B}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NuGet.Tasks", "src\NuGet.Tasks\NuGet.Tasks.csproj", "{020ED083-4076-4711-A52B-2F89EA884F9B}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RuntimeStoreVerifier", "src\RuntimeStoreVerifier\RuntimeStoreVerifier.csproj", "{1B5856CC-E597-4E97-9C95-DEAC80CE724A}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "WinTrust.Sources", "WinTrust.Sources", "{B99861A9-F594-4FCF-A09F-9730A3FF9436}"
+	ProjectSection(SolutionItems) = preProject
+		src\WinTrust.Sources\WinTrust.cs = src\WinTrust.Sources\WinTrust.cs
```

Comment:
Nit: can you put this `shared/` instead of `src/`? WinTrust.Sources isn't a csproj, right?